addons/website_blog/views/website_blog_templates.xml

```diff
@@ -136,12 +135,18 @@ list of filtered posts (by date or tag).
                 </div>
             </div>
         </t>
-        <t t-else="">
-            <div class="h1 my-4 o_not_editable" style="text-align:center;">Our Latest Posts</div>
-        </t>
     </xpath>
 </template>
 
+<!-- (Option) Blog: Show latest-post as top banner : 'Regular' design -->
+<template id="opt_general_blog_cover" name="General Blog Cover">
+    <section class="d-flex justify-content-center align-items-center py-4">
+        <div class="text-center">
+            <h1>Our latest post</h1>
+            <h2 class="h5-fs">Check out our latest posts</h2>
```

Comment:
- Is the second sentence useful? It is the same as the first one with "Check out" in front of it. Maybe add checkout in the first one?
- If useful, then I am not sure it should be a h2